docs/tutorial/using-backup-with-syclladb.md

```diff
@@ -0,0 +1,235 @@
+    vald-backup-gateway-68c8b4ffd4-df8zp       1/1     Running     0          6m56s
+    vald-backup-gateway-68c8b4ffd4-dmwrd       1/1     Running     0          6m56s
+    vald-backup-gateway-68c8b4ffd4-nm8f7       1/1     Running     0          7m12s
+    vald-discoverer-7f9f697dbb-q44qh           1/1     Running     0          7m11s
+    vald-lb-gateway-6b7b9f6948-4z5md           1/1     Running     0          7m12s
+    vald-lb-gateway-6b7b9f6948-68g94           1/1     Running     0          6m56s
+    vald-lb-gateway-6b7b9f6948-cvspq           1/1     Running     0          6m56s
+    vald-manager-backup-5fb5f8dc7-h22sv        1/1     Running     0          7m12s
+    vald-manager-backup-5fb5f8dc7-ncrw4        1/1     Running     0          6m56s
+    vald-manager-backup-5fb5f8dc7-nzbkh        1/1     Running     0          6m56s
+    vald-manager-compressor-78bf64459f-27ckg   1/1     Running     0          6m56s
+    vald-manager-compressor-78bf64459f-9kl9b   1/1     Running     0          7m12s
+    vald-manager-compressor-78bf64459f-dkx24   1/1     Running     0          6m56s
+    vald-manager-index-74c7b5ddd6-jrnlw        1/1     Running     0          7m12s
+    vald-meta-747f757bbb-9v5xz                 1/1     Running     0          7m12s
+    vald-meta-747f757bbb-mpwqp                 1/1     Running     0          6m56s
+    vald-meta-gateway-8c5f55dd-8fsch           1/1     Running     0          6m56s
+    vald-meta-gateway-8c5f55dd-sdd5q           1/1     Running     0          7m12s
+    vald-meta-gateway-8c5f55dd-vfkn6           1/1     Running     0          6m56s
+    ```
+
+    </details>
+
+### Run using example code
+
+This chapter shows how to perform a search action in Vald with fashion-mnist dataset.
+
+1. Port Forward
+
+    At first, port-foward is required to make request from your local environment possible.
+
+    ```bash
+    kubectl port-forward deployment/vald-meta-gateway 8081:8081
+    ```
+
+1. Download dataset
+
+    Download [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) that is used as dataset for indexing and search query.
+
+    ```bash
+    # move to working directory
+    cd example/client
+
+    # download fashion-mnist testing dataset
+    wget http://ann-benchmarks.com/fashion-mnist-784-euclidean.hdf5
+    ```
+
+1. Running example
+
+    We use [`example/client/main.go`](https://github.com/vdaas/vald/blob/master/example/client/main.go) to run the example.<br>
+    This exapmle will insert and index 400 vectors into the Vald from the fashion-mnist dataset via grpc.
+    And then after waiting for indexing, it will request for searching nearest vector at 10 times.
+    You will get the 10 nearest neighbor vectors for each search query.<br>
+    Run example codes by executing below command.
+
+    ```bash
+    # run example
+    go run main.go
+    ```
+
+    The datailed explanation of exapmle code is shown in [Get Started](../tutorial/get-started.md#Running-Example-Code)
+
```

Comment:
I think it is better to include command to check if the data is backup successfully

Reply:
I'll stock as future task